lib/ui/shared/bottom_sheets/thread_options_bottomsheet/thread_option_bottomsheet_viewmodel.dart

```diff
@@ -1,3 +1,20 @@
+import 'package:flutter/material.dart';
+import 'package:hng/ui/shared/dialogs/remind_me_dialog/remind_me_dialog.dart';
 import 'package:stacked/stacked.dart';
 
-class ThreadOptionsBottomSheetViewModel extends BaseViewModel {}
\ No newline at end of file
+class ThreadOptionsBottomSheetViewModel extends BaseViewModel {
+  remindMe(context) async {
+    return await showDialog(
```

Comment:
You're supposed to use the dialog service to call this, no context is needed.
Also material shouldn't be imported in the view model